app/models/form_profile.rb

```diff
@@ -286,16 +286,17 @@ def vet360_mailing_address
     vet360_contact_info&.mailing_address
   end
 
-  def va_profile_address_hash
-    user.va_profile&.address &&
-      {
-        street: user.va_profile.address.street,
-        street2: nil,
-        city: user.va_profile.address.city,
-        state: user.va_profile.address.state,
-        country: user.va_profile.address.country,
-        postal_code: user.va_profile.address.postal_code
-      }
+  def user_address_hash
+    return unless user.address
```

Comment:
Now you don't even need this line! Because the `user.address` object always returns a hash, it'll always have the `street`, `city`, etc fields on it, even if it's nil